### PR TITLE
docs(doc): add description in README.md

### LINKED ISSUE #11: Add documentation in README.md

### LINKED ISSUE #16: Add comments to improve documentation
According to [this](https://github.com/pseudomuto/protoc-gen-doc#writing-documentation), we can add comments to enrich the documentation.

### PR DESCRIPTION
**Pull request by bochengyang**
Because

- enrich more content to README.md to tell users what this repository is

This commit

- add description in README.md
- close #11

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,12 @@
...
+This repository is the interface definitions of public Instill AI APIs that support both REST and gRPC protocols. We think this repository can provide a better understanding of Instill AI APIs by reading the original interface definitions to help you utilize them more efficiently. You can also use these definitions with open source tools to generate client libraries, documentation, and other artifacts.
+
+## Overview
+
+Instill AI APIs use Protocol Buffers version 3 (proto3) as our Interface Definition Language (IDL) to define the API interface and the structure of the payload messages. The same interface definition is used for both REST and RPC versions of the API, which can be accessed over different wire protocols
+
+There are several ways of accessing Instill AI APIs:
+
+1. JSON over HTTP: You can access all Instill AI APIs directly using JSON over HTTP
+2. Protocol Buffers over gRPC: You can also use GRPC, which is a high-performance binary RPC protocol over HTTP/2, to access Instill AI APIs. It offers many useful features, including request/response multiplex and full-duplex streaming.
```
Comment (xiaofei-du) [inline README.md:12]:
```suggestion
# Visual Data Preparation (VDP) Protobufs

This repository is the interface definitions of [Visual Data Preparation (VDP) APIs](https://github.com/instill-ai/vdp) that support both REST and gRPC protocols. You can also use these definitions with open source tools to generate client libraries, documentation, and other artifacts.

## Overview

VDP APIs use Protocol Buffers version 3 (proto3) as our Interface Definition Language (IDL) to define the API interface and the structure of the payload messages. The same interface definition is used for both REST and RPC versions of the API, which can be accessed over different wire protocols

There are several ways of accessing VDP APIs:

1. JSON over HTTP: You can access directly using JSON over HTTP
2. Protocol Buffers over gRPC: You can also use GRPC, which is a high-performance binary RPC protocol over HTTP/2. It offers many useful features, including request/response multiplex and full-duplex streaming.
```

**Review by xiaofei-du — APPROVED** [review]
I have made some small changes. Other than that, LGTM 👍

File: README.md (hunk 1 above)
Comment (pinglin) [inline README.md:12]:
```suggestion
2. Protocol Buffers over gRPC: You can also use gRPC, which is a high-performance binary RPC protocol over HTTP/2, to access Instill AI APIs. It offers many useful features, including request/response multiplex and full-duplex streaming.
```

File: README.md [hunk 2]
```diff
@@ -1 +1,12 @@
-# protobufs
+# Instill AI  APIs
+
+This repository is the interface definitions of public Instill AI APIs that support both REST and gRPC protocols. We think this repository can provide a better understanding of Instill AI APIs by reading the original interface definitions to help you utilize them more efficiently. You can also use these definitions with open source tools to generate client libraries, documentation, and other artifacts.
+
+## Overview
+
+Instill AI APIs use Protocol Buffers version 3 (proto3) as our Interface Definition Language (IDL) to define the API interface and the structure of the payload messages. The same interface definition is used for both REST and RPC versions of the API, which can be accessed over different wire protocols
```
Comment (pinglin) [inline README.md:7]:
```suggestion
Instill AI APIs use Protocol Buffers version 3 (proto3) as our Interface Definition Language (IDL) to define the API interface and the structure of the payload messages. The same interface definition is used for both REST (via [gRPC-Gateway](https://github.com/grpc-ecosystem/grpc-gateway)) and RPC versions of the API, which can be accessed over different wire protocols
```

Comment (bochengyang) [reply README.md]: Fixed in this [commit](https://github.com/instill-ai/protobufs/pull/20/commits/3f02c41c242b906bd0a39c9669057970d2150c49)

File: README.md (hunk 1 above)
Comment (bochengyang) [reply README.md]: Fixed in this [commit](https://github.com/instill-ai/protobufs/pull/20/commits/3f02c41c242b906bd0a39c9669057970d2150c49)

Comment (bochengyang) [reply README.md]: Fixed in this [commit](https://github.com/instill-ai/protobufs/pull/20/commits/3f02c41c242b906bd0a39c9669057970d2150c49)

**Review by pinglin — APPROVED** [review]
LGTM

**Pull request merged**